play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -2,645 +2,696 @@ Core classes:
+	you must choose to either use your cover's defense rating or the defense provided 
+	by this feat, you cannot use both. Gain +1 additional defense at level 10. This
+	feat's defense bonus DOES stack with crouching or prone if you aren't behind cover.
+	You can have a MAX of 5 Miss Chance Defense from this feat.
+	*(If dual wielding, use the larger of the two Weapon Reflexes).
+[THE FOLLOWNG PASSIVES ONLY APPLY TO PISTOLS]
+Choose two bonuses and one weakness:
+    - Bonuses: +10 Dam, +8m R3 Range Bracket, -1 Miss Chance, +1 rds in mag, +2 mags carried;
+    - Weaknesses: -10 Dam, -8m R3 Range Bracket, +1 Miss Chance, -1 rds in mag, -2 mags carried;
+Weapon Proficiencies: Pistols.
+Armor Proficiencies: Concealed, Recon, Light.
+Starting Wealth: $850 + 100*Luck + 1125
 Feats:
-----------------------
-Heightened awareness -
-Being aware comes with your level of training. You cannot be caught flat-footed
-, but opponents can still get surprise rounds against you.
-|
-|
-|
-V
-You are omni-aware to any attacks made against you or allies. Whenever an 
-enemy attempts to make an attack of opportunity on you or any allies within eye
--sight, you gain an attack of opportunity on them.
-----------------------
-Enhanced Gun-Kata -
-You are a disciple of the art of close-range pistol fighting. Applies to any 
-target within 15m. On a successful hit, you may continue rolling hits until 
-you fail to hit or run out of rounds in your magazine. You must make each 
-successive hit on a different target. THIS MEANS ONE HIT PER TARGET PER KATA.
-|
-|
-|
-V
-You are a master of pistols. Applies to an target within 35m. On a successful 
-hit, you may continue rolling hits until you fail to hit. if you run out of 
-rounds, you may reload mid-kata by passing a DC 75% DEX check. You may hit a 
-target multiple times, but there must be one shot between shots fired at a 
-single target. (Ask GM).
-----------------------
-
-"Assassin" - The galaxy is a dangerous place for those who would believe their 
-power or influence are limitless. The assassin lurks in the shadows, either as 
-a lone force of justice, or as part of a larger organization. They have 
-rightfully earned the title of Galaxy''s greatest Hand to Hand Fighters.
-Passives: No penalty for engaging in Grappling maneuvers. 
-Assassins wear a Shadow Cloak, which allows them to become a mere silhouette. 
-When in partial darkness or darker, the static DC increases by 10.
-NOTICE: SEE BASIC RULES FOR ADDITIONAL CLOAKING ATTRIBUTES
-Weapon Proficiencies: Pistol, Hand to Hand Expert
-Armour Proficiencies: Concealed, Recon, Light, Cloaking.
-Starting Wealth: $1700+100*Luck - 1300
+	----------------------
+	Heightened Alertness -
+	Being alert comes with your level of training. Gain 10 Reflex. You cannot be caught 
+	flat-footed (you cannot be penalized for being surprised), but opponents can still 
+	get surprise rounds against you.
+	----------------------
+	Protective Awareness - Requires a minimum Reflex of 24
```

Comment:
Gives more value to Heightened Alertness, which adds 10 reflex.
Also, Protective Awareness is extremely powerful, so having a high minimum will either force the player to min-max very hard or wait until level 10 so they can first acquire Heightened Alertness.

Reply:
Oh, and the player will likely be vying for high reflex already since that gives them a passive miss-chance defense. So I think it's prudent to set the minimum to be quite high so the ability isn't a "gimme" but requires more thought

Reply:
@bleehu

Reply:
Thanks for letting me know the thinking. All good.